lcd/circuitpython_i2c_lcd_test.py

```diff
@@ -0,0 +1,80 @@
+"""Implements a HD44780 character LCD connected via PCF8574 on I2C in CircuitPython.
+   This was tested with the Adafruit Trinket M0
+   https://www.adafruit.com/product/3500"""
+
+import board
+import busio
+from time import sleep, monotonic
+from circuitpython_i2c_lcd import I2cLcd
+
+# The PCF8574 has a jumper selectable address: 0x20 - 0x27
+DEFAULT_I2C_ADDR = 0x27
+
+def test_main():
+	"""Test function for verifying basic functionality."""
+	print("Running test_main")
+	i2c = busio.I2C(board.SCL, board.SDA)
```

Comment:
This file should use spaces rather than tabs for indentation (4 spaces is consistent with the remaining files)